CSS/quiz-home.css

```diff
@@ -0,0 +1,81 @@
+main{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    background-color: var(--section-background-color);
+    width: 100%;
+    min-height: 100vh;
+}
+.footer-div{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    height: 5rem;
+    background-color: var(--background-color);
+}
+.main-content-area{
+    margin-top: 5rem;
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    align-items:center;
+    gap: 1rem;
+}
+.paragraph{
+    font-size: 1.5rem;
+    width: 100%;
+    color: var(--background-color);
+}
+.text-div{
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+}
+.hero-img-div{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 30rem;
+    height: 20rem;
+}
+.hero-img{
+    width: 30rem;
+}
+.footer-text{
+    color: var(--headline-color);
+}
+.nav-bar{
+    padding: 1rem 3rem 1rem 3rem;
```

Comment:
we can use the shorthand property ``padding: top/bottom | left/right``

Reply:
oh ya i just forget to use that thanks for pointing out updated